taxonomy-editor-frontend/src/pages/editentry/index.jsx

```diff
@@ -4,6 +4,7 @@ import FetchRelations from "./FetchRelations";
 import useFetch from "../../components/useFetch";
 import ListAllEntryProperties from "./ListAllEntryProperties";
 import ListAllOtherProperties from "./ListAllOtherProperties";
+import AccumulateAllComponents from "./AccumulateAllComponents";
 
 const EditEntry = () => {
     let url = 'http://localhost:80/'

```

Comment:
what about passing this to a constants.js file? or maybe even an env variable?, thus you don't need to pass it all the way down. (don't know if this is the case, but for [ create-react-app ](https://create-react-app.dev/docs/adding-custom-environment-variables/#adding-development-environment-variables-in-env) )

Reply:
I can create a constants.js file, but I can't avoid passing the url since the URL for accessing each entry is dependant on it's id.